src/govuk/components/pagination/pagination.yaml

```diff
@@ -0,0 +1,169 @@
+params:
+- name: previous
+  type: object
+  required: false
+  description: A link to the previous page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The previous page link text. Defaults to 'Previous page', where 'page' is visually hidden.
+  - name: label
```

Comment:
Can we use `labelText` here (and for next) so that we can introduce `labelHtml` later if we find there's a need for it?